src/components/Firestore/Requests/RequestPath/index.tsx

```diff
@@ -0,0 +1,170 @@
+/**
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import '../index.scss';
+
+import { IconButton } from '@rmwc/icon-button';
+import { Tooltip } from '@rmwc/tooltip';
+import React, { useEffect, useRef, useState } from 'react';
+
+// Copies path without spaces to clipboard
+// and triggers copy notification (SnackBar)
+function copyPathToClipboard(
+  resourcePath: string,
+  setShowCopyNotification: (value: boolean) => void
+) {
+  navigator.clipboard
+    .writeText(resourcePath.replace(/\s/g, ''))
+    .then(() => setShowCopyNotification(true))
+    .catch(() => setShowCopyNotification(false));
```

Comment:
Consider showing an error when this throws (or if `navigator.clipboard` is unsupported)

Reply:
Can I use the error-reason that comes from the rejected Promise? or would it be more helpful to display a custom string like: `'Could not copy request-path to clipboard` ?

![image](https://user-images.githubusercontent.com/34299428/106037940-efd7e400-609c-11eb-949f-aa8cecf7d949.png)